lxd/device/device_utils_disk.go

```diff
@@ -256,7 +256,12 @@ func diskCephfsOptions(clusterName string, userName string, fsName string, fsPat
 	fsOptions := fmt.Sprintf("name=%v,secret=%v,mds_namespace=%v", userName, secret, fsName)
 	srcpath := ""
 	for _, monAddress := range monAddresses {
-		srcpath += fmt.Sprintf("%s:6789,", monAddress)
+		// Add the default port to the mon hosts if not already provided
+		if strings.Contains(monAddress, ":6789") {
+				srcpath += fmt.Sprintf("%s,", monAddress)
+		} else {
+				srcpath += fmt.Sprintf("%s:6789,", monAddress)
+    }
```

Comment:
Indent looks quite off, can you run `go fmt ./...` to fix this?

Reply:
Fixed